contrib/http_servlet/src/main/java/io/opencensus/contrib/http/servlet/OcHttpServletFilter.java

```diff
@@ -145,15 +145,11 @@ public void init(FilterConfig filterConfig) throws ServletException {
       extractor = new OcHttpServletExtractor();
     }
 
-    obj = context.getAttribute(OC_PUBLIC_ENDPOINT);
+    String publicEndVal = context.getInitParameter(OC_PUBLIC_ENDPOINT);
     if (obj != null) {
-      if (obj instanceof Boolean) {
-        publicEndpoint = (Boolean) obj;
-      } else {
-        throw new ServletException(EXCEPTION_MESSAGE + OC_PUBLIC_ENDPOINT);
-      }
+        publicEndpoint = Boolean.parseBoolean(publicEndVal);
     } else {
-      publicEndpoint = true;
+      publicEndpoint = false;
```

Comment:
This is already fixed in #1781. Please rebase your change

Reply:
Rebased in latest push